integration-tests/chaos/ocr2vrf_chaos_test.go

```diff
@@ -0,0 +1,276 @@
+package chaos
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/onsi/gomega"
+	"github.com/smartcontractkit/chainlink-env/chaos"
+	"github.com/smartcontractkit/chainlink-env/environment"
+	a "github.com/smartcontractkit/chainlink-env/pkg/alias"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/ethereum"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/utils"
+	ocr2vrftypes "github.com/smartcontractkit/ocr2vrf/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/actions/ocr2vrf_actions"
+
+	networks "github.com/smartcontractkit/chainlink/integration-tests"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions"
+	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/config"
+	"github.com/smartcontractkit/chainlink/integration-tests/contracts"
+
+	"github.com/rs/zerolog/log"
+)
+
+var (
+	defaultOCR2VRFSettings = map[string]interface{}{
+		"replicas": "6",
+		"toml": client.AddNetworkDetailedConfig(
+			config.BaseOCR2VRFTomlConfig,
+			config.DefaultOCR2VRFNetworkDetailTomlConfig,
+			networks.SelectedNetwork),
+	}
+
+	defaultOCR2VRFEthereumSettings = &ethereum.Props{
+		NetworkName: networks.SelectedNetwork.Name,
+		Simulated:   networks.SelectedNetwork.Simulated,
+		WsURLs:      networks.SelectedNetwork.URLs,
+	}
+
+	linkEthFeedResponse          = big.NewInt(1e18)
+	linkFundingAmount            = big.NewInt(100)
+	beaconPeriodBlocksCount      = big.NewInt(3)
+	ethFundingAmount             = big.NewFloat(1)
+	numberOfRandomWordsToRequest = uint16(1)
+	confirmationDelay            = big.NewInt(1)
+	subscriptionID               = uint64(1)
+	//keyID can be any random value
+	keyID = "aee00d81f822f882b6fe28489822f59ebb21ea95c0ae21d9f67c0239461148fc"
+
```

Comment:
I also was thinking about this and could not decide, since I prefer to see the input variables inside a test...but also agree that i can encapsulate at least default variables, will update

Reply:
done